src/main/java/com/bullhornsdk/data/model/entity/core/standard/ClientCorporation.java

```diff
@@ -38,8 +38,8 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonRootName(value = "data")
 @JsonPropertyOrder({ "id", "address", "annualRevenue", "billingAddress", "billingContact", "billingFrequency", "billingPhone",
-		"branch", "businessSectorList", "childClientCorporations", "clientContacts", "companyDescription", "companyURL",
-		"competitors", "culture", "customDate1", "customDate2", "customDate3", "customFloat1", "customFloat2", "customFloat3",
+        "branch", "businessSectorList", "certifications", "certificationGroups", "childClientCorporations", "clientContacts", "companyDescription",
```

Comment:
should this be requirements or certifications?

Reply:
Should be requirements. Fixed